src/guide/views/dropdown/DropdownExample.jsx

```diff
@@ -76,6 +94,16 @@ export default class DropdownExample extends Component {
           />
         </Example>
 
+        <Example title="With group">
+          <DropdownGroup
+            labelProvider={this.dropdownLabelProvider}
+            options={this.dropdownItemsGroup}
+            optionsClass={this.dropdownItemsClass}
+            onSelect={this.onSelectDropdownOption}
+            optionLabelProvider={this.dropdownOptionLabelProvider}
+          />
+        </Example>
```

Comment:
Selecting an option doesn't change the label, but changes the content of the other dropdowns.